.travis.yml

```diff
@@ -43,4 +43,11 @@ before_script:
   - 'if [ "$LINT" == "true" ]; then npm run lint; fi'
 
 script:
-  - npm run travis
\ No newline at end of file
+  - npm run travis
+
+after_success:
+  # run automated release process with semantic-release
+  - if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_EVENT_TYPE" == "push" && "$TRAVIS_NODE_VERSION" == "12" ]]; then
+      npm i --no-save semantic-release@15 @semantic-release/changelog@3 @semantic-release/git@7;
+      semantic-release;
```

Comment:
Is `semantic-release` installed globally? If not I think we need to call `npm run semantic-release`.

Reply:
I just used copy/paste from the working case in karma-jasmine:
https://github.com/karma-runner/karma-jasmine/blob/master/.travis.yml#L21